lib/broccoli/ember-app.js

```diff
@@ -217,8 +205,12 @@ EmberApp.prototype._initOptions = function(options) {
   var publicPath = this._resolveLocal('public');
   var publicTree = existsSync(publicPath) ? new WatchedDir(publicPath) : null;
 
-  this.options = defaultsDeep(options, {
-    babel: babelOptions,
+  var detectedOptions = {
```

Comment:
Why are we calling this `detectedOptions`? It feels like it's an extension to the default option set. The thing is that we're looking at project setup to figure these out. I just want a better name or a comment.